go.mod

```diff
@@ -58,3 +58,5 @@ replace (
 	gopkg.in/couchbaselabs/gocbconnstr.v1 => github.com/couchbaselabs/gocbconnstr v1.0.5
 	k8s.io/client => github.com/kubernetes-client/go v0.0.0-20190928040339-c757968c4c36
 )
+
+replace github.com/dapr/components-contrib => ../components-contrib
```

Comment:
Is it for test?

Reply:
Yes because the changes to the components are not in the latest branch (it is in master).

        github.com/dapr/components-contrib/bindings/graphql: module github.com/dapr/components-contrib@latest found (v1.2.0), but does not contain package github.com/dapr/components-contrib/bindings/graphql